DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -323,6 +343,47 @@ public void UpdateCourseOptions(CourseOptions courseOptions, int customisationId
             return courseDataService.GetCourseValidationDetails(customisationId, centreId)?.CourseCategoryId;
         }
 
+        public int CreateNewCentreCourse(
+            int centreId,

```

Comment:
I would be tempted to create a new class for this rather than lots of arguments.
In saying that if we have great certainty that this list of arguments was not going to change we may not gain much in that respect.
Although it may make the code slightly easier to read and test.
We do already have a `Course` class but that appears to not be useful in this context. 
If we had a new class it could be called something along the lines of `Customisation` given that is what is being inserted.

Reply:
I agree, I've creates a class Customisation to contain the information. Makes the code a lot cleaner to read